reference-architecture/azure-ansible/testcases/test_pv.sh

```diff
@@ -0,0 +1,8 @@
+oc new-project demo
```

Comment:
not at this point, as I want to tech freeze and get doc finished. Then can add lots of value I have already got hooks for 3.5 release.